src/main/java/com/northconcepts/datapipeline/examples/kafka/ReadAndWriteKafkaEvents.java

```diff
@@ -0,0 +1,39 @@
+package com.northconcepts.datapipeline.examples.kafka;
+
+import com.northconcepts.datapipeline.core.DataReader;
+import com.northconcepts.datapipeline.core.DataWriter;
+import com.northconcepts.datapipeline.csv.CSVReader;
+import com.northconcepts.datapipeline.csv.CSVWriter;
+import com.northconcepts.datapipeline.job.Job;
+import com.northconcepts.datapipeline.kafka.KafkaReader;
+import com.northconcepts.datapipeline.kafka.KafkaWriter;
+
+import java.io.File;
+import java.util.Properties;
+
+public class ReadAndWriteKafkaEvents {
+
+    private static String BOOTSTRAP_SERVERS = "localhost:9092,another.host:port";
+    private static String GROUP_ID = "group_id";
+
+    public static void main(String[] args) {
+        DataReader reader = new CSVReader(new File("data/input/jewelry.csv"))
+            .setAllowMultiLineText(true)
+            .setFieldNamesInFirstRow(true);
+
+        Properties writerProps = new Properties();
+        writerProps.put("bootstrap.servers", BOOTSTRAP_SERVERS);
+        DataWriter writer = new KafkaWriter(writerProps, "jewelry");
+
+        Job.run(reader, writer);
```

Comment:
Do we need to run any of these job asynchronously -- like we had to for the tests?

Reply:
In the tests we started the writing first and ran it asynchronously. Also increased the poll timeout for the reader so that `Thread.sleep` would no longer be necessary. Updated this example